sdk/Lusid.Sdk.Tests/tutorials/Instruments/DemoEquityOption.cs

```diff
@@ -0,0 +1,181 @@
+                scope: scope,
+                code: recipeCode,
+                market: new MarketContext(
+                    new List<MarketDataKeyRule> {},
+                    options: new MarketOptions(defaultSupplier: "Lusid", defaultScope: scope, defaultInstrumentCodeType: "RIC")
+                    ),
+                pricing: new PricingContext(options: pricingOptions)
+            );
+            _testDataUtilities.UpsertRecipe(recipe);
+
+            // define the metrics that we wish to return
+            string ValuationDateKey = "Analytic/default/ValuationDate";
+            string InstrumentTag = "Analytic/default/InstrumentTag";
+            string HoldingPvKey = "Holding/default/PV";
+            var valuationSpec = new List<AggregateSpec>
+            {
+                new AggregateSpec(ValuationDateKey, AggregateSpec.OpEnum.Value),
+                new AggregateSpec(InstrumentTag, AggregateSpec.OpEnum.Value),
+                new AggregateSpec(HoldingPvKey, AggregateSpec.OpEnum.Value)
+            };
+
+            // choose valuation dates
+            var valuationSchedule = new ValuationSchedule(effectiveAt: _demoEffectiveAt);
+
+            // construct and perform valuation request
+            var instruments = new List<WeightedInstrument> {new WeightedInstrument(1, "some-holding-identifier", option)};
+            var inlineValuationRequest = new InlineValuationRequest(
+                recipeId: new ResourceId(scope, recipeCode),
+                metrics: valuationSpec,
+                sort: new List<OrderBySpec> {new OrderBySpec(ValuationDateKey, OrderBySpec.SortOrderEnum.Ascending)},
+                valuationSchedule: valuationSchedule,
+                instruments: instruments);
+            var valuation = _aggregationApi.GetValuationOfWeightedInstruments(inlineValuationRequest);
+            Assert.That(valuation, Is.Not.Null);
+            Assert.That(valuation.Data.Count, Is.EqualTo(instruments.Count));
+
+            var pv = valuation.Data[0][HoldingPvKey];
+            Assert.That(pv, Is.Positive);
+            Console.WriteLine($"Computed pv of {pv} at time {_demoEffectiveAt:O}");
+        }
+
+        [Test]
+        public void DemoEquityOptionCashFlows()
+        {
+            var scope = "DemoEquityOptionCashFlows";
+
+            // upsert option
+            var option = InstrumentExamples.CreateExampleEquityOption();
+            string uniqueId = "id-equityOption-1";
+            var response = _testDataUtilities.UpsertOtcToLusid(option, "some-name-for-this-equityOption", uniqueId);
+            var luid = response.Values.First().Value.LusidInstrumentId;
+
+            // for equity option cashflows, we need the following market data to determine intrinsic value
+            _testDataUtilities.CreateAndUpsertSimpleQuote(scope, "ACME", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 110m, "USD", _demoEffectiveAt);
+
+            // create a new portfolio and add the option to it via a transaction
```

Comment:
Have a look, think the style here is to (also) cap the first word

Reply:
Personally I don't see the point, but done. I have now capitalised the words CREATE, UPSERT, DEFINE, CHOOSE, CONSTRUCT, and PERFORM. 4 of these are synonyms lol

Reply:
Well FBN style.....a la coding style (internally or externally)